### ISSUE TITLE
model-mesh serving runtimes images set incorrectly

### ISSUE THREAD

**Issue by shalberd** [opendatahub-io/odh-manifests#809]
**Describe the bug**

Manifest branch: master
Date: 6 June 2023

it almost seems the model-mesh/odh-modelmesh-controller/runtimes directory is never evaluated and process by kustomize.

Is that intended? That would explain why the inconsistencies have received so little attention. The error
"well-defined vars that were never replaced: odh-openvino" and in general var references and lookups are not a problem in all other components except model-mesh.

for model-mesh serving runtimes controller, for ovms/openvino there is currently a mix of
[referenced](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/odh-modelmesh-controller/runtimes/ovms-1.x.yaml#L37)

[kustomize parameters](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/base/params.env#L5)  for ovms/openvino and 

and [newName and imageTag](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/odh-modelmesh-controller/runtimes/kustomization.yaml#L29) 

aside from ovms, e.g. for mlserver should the image-field-names not be equal to what is specified in kustomization.yaml under images? Currently, there is an [image-field value of e.g. mlserver-0:replace in the kustomization.yaml](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/odh-modelmesh-controller/runtimes/mlserver-0.x.yaml#L41) and a value to look for named [mlserver-0 in images section of kustomization.yaml](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/odh-modelmesh-controller/runtimes/kustomization.yaml#L25). That doesn't work, I think.

That leads to the following error in kustomize:

```
s$ kustomize build  ./model-mesh/base > model-mesh-all.yaml
2023/06/01 11:38:02 well-defined vars that were never replaced: odh-openvino
```

I think that is because the images section in kustomization.yaml first tries to apply the image value.
Basically, in that kustomization.yaml, there is a lot wrong.

From a maintainer perspective: 

What is the preferred way for setting the path, name, and tag/digest: kustomization images section or parameters?

Related topic, in odh-dashboard:

https://github.com/opendatahub-io/odh-dashboard/pull/1210/files#diff-a1084b104b138554b5e938c54cb6ac25200adcd675bf4d1fb89a02a6b91d5bfa

**Steps To Reproduce**

**Expected behavior**

**Workaround (if any)**
Any manual steps that allow you to resolve the issue

**Open Data Hub Version**
Please attach relevant kfdef manifest if applicable

**OpenShift Version**
Version #: 
Provider (Baremetal, OpenStack, RHV, AWS, OKD, CodeReady Containers, ...):

**Additional context**

**Comment by shalberd** [comment]
@Jooho I think my input does not apply anymore / does not pose a problem anymore with your recent changes.

**Comment by shalberd** [comment]
on odh community / upstream, ClusterServingRuntime CR is not supported, only Template CR used via odh-dashboard:

https://github.com/opendatahub-io/odh-dashboard/blob/7939300992d9ec7198c54fe02b8b73503265b40f/frontend/src/api/k8s/templates.ts#L40

Closing issue
